default_app/renderer.ts

```diff
@@ -27,8 +27,8 @@ function initialize () {
   }
 
   document.querySelector<HTMLAnchorElement>('.electron-version')!.innerText = `Electron v${process.versions.electron}`
-  document.querySelector<HTMLAnchorElement>('.chrome-version')!.innerText = `Chromium v${process.versions.chrome}`
-  document.querySelector<HTMLAnchorElement>('.node-version')!.innerText = `Node v${process.versions.node}`
+  document.querySelector<HTMLAnchorElement>('.chrome-version')!.innerText = `<a href="https://www.chromium.org/v${process.versions.chrome}">Chromium v${process.versions.chrome}</a>`
+  document.querySelector<HTMLAnchorElement>('.node-version')!.innerText = `<a href="https://nodejs.org/docs/v${process.versions.node}">Node v${process.versions.node}</a>`
```

Comment:
Same thing here: `innerHTML` is required. Also, I'm noticing that a direct link constructed this way leads to [a folder listing](https://nodejs.org/docs/v10.0.0/), so we probably want to add [an `/api` at the end](https://nodejs.org/docs/v10.0.0/api).

```suggestion
  document.querySelector<HTMLAnchorElement>('.node-version')!.innerHTML = `<a href="https://nodejs.org/docs/v${process.versions.node}/api">Node v${process.versions.node}</a>`
```

Reply:
Sounds good 👍